fv3core/stencils/dyn_core.py

```diff
@@ -149,19 +153,17 @@ def compute(state, comm):
 
         # TODO: Is really just a column... when different shapes are supported
         # perhaps change this.
-        state.dp_ref = utils.make_storage_from_shape(
-            state.ak.shape, grid.default_origin()
-        )
-        state.zs = utils.make_storage_from_shape(state.ak.shape, grid.default_origin())
+        state.dp_ref = utils.make_storage_from_shape(state.ak.shape, grid.full_origin())
+        state.zs = utils.make_storage_from_shape(state.ak.shape, grid.full_origin())
         dp_ref_compute(
             state.ak,
             state.bk,
             state.phis,
             state.dp_ref,
             state.zs,
             rgrav,
-            origin=grid.default_origin(),
-            domain=grid.domain_shape_standard(add=(0, 0, 1)),
+            origin=grid.full_origin(),
+            domain=grid.domain_shape_full(),
```

Comment:
I think you need an 'add=(0, 0, 1)' here - this may be why DynCore is failing the tests

Reply:
That fixed it, thanks!